html-css/week2/index.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title>Minichimp Help</title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="Styles.css">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.7.2/css/all.min.css" integrity="sha512-Evv84Mr4kqVGRNSgIGL/F/aIDqQb7xQ2vcrdIwxfjThSH8CSR7PBEakCr51Ck+w+/U6swU2Im1vVX0SVk9ABhg==" crossorigin="anonymous" referrerpolicy="no-referrer" />       
+        <script type="module" src="https://unpkg.com/ionicons@7.1.0/dist/ionicons/ionicons.esm.js"></script>
+        <script nomodule src="https://unpkg.com/ionicons@7.1.0/dist/ionicons/ionicons.js"></script>
+    </head>
+    <body>
+        <header class="header">
+            <div class="searchbar">
+                <form action="" class="search">
+                    <i class="fa-solid fa-magnifying-glass"></i>
+                    <input type="text" placeholder="How ...">
+                    <i class="fa-solid fa-xmark"></i>
+                </form>
+            </div>
+            <div>
+                 <ul class="navbar">
+                    <li><a class="navbar" href="#Campains">Campains</a></li>
+                    <li><a class="navbar" href="#Contact">Contact</a></li>
+                    <li><a class="navbar" href="#Help Article">Help Article</a></li>
+                </ul>
+            </div>
+        </header>
+        <section>

```

Comment:
The section tag is mostly used to separate different sections of a page. For example, here all the results found would be one section, and then you could separate each result with a div.